src/r_util.cpp

```diff
@@ -521,6 +554,7 @@ namespace rhost {
             { "Microsoft.R.Host::Call.destroy_blob", (DL_FUNC)destroy_blob, 1 },
             { "Microsoft.R.Host::Call.get_file_lock_state", (DL_FUNC)get_file_lock_state, 1 },
             { "Microsoft.R.Host::Call.fetch_file", (DL_FUNC)fetch_file, 1 },
+            { "Microsoft.R.Host::Call.save_project", (DL_FUNC)save_project, 4 },
```

Comment:
@int19h Do you think we need to provide any additional error information?

---
In reply to: [74654678](https://github.com/Microsoft/R-Host/pull/78#discussion_r74654678) [](ancestors = 74654678)